nuxeo-drive-client/nxdrive/osi/windows.py

```diff
@@ -20,9 +20,26 @@ def get_menu_key(self):
         return self.get_menu_parent_key() + '\\command'
 
     def _delete_reg_value(self, reg, path, value):
-        key = _winreg.OpenKey(reg, path, 0, _winreg.KEY_ALL_ACCESS)
-        _winreg.DeleteValue(key, value)
+        try:
+            key = _winreg.OpenKey(reg, path, 0, _winreg.KEY_ALL_ACCESS)
+        except Exception, e:
+            # Sometimes the app is installed, but not run 
+            # Hence the path argument is invalid
+            # It throws WindowsError 2: The system cannot find the file specified
+            # The uninstallation doesn't complete if exception is not caught
+            print e
```

Comment:
I suppose the print was used because log is not available while uninstall, do you think we can get rid of them ?